kratos/tests/test_function_parser_utility.py

```diff
@@ -169,6 +169,11 @@ def test_ApplyFunctionToNodesUtilityTimeEvolutionCTernaryFail(self):
         with self.assertRaisesRegex(Exception, 'Parsing error in function: 1.5 if t<2.0 3.0 if defined, but not else'):
             KM.GenericFunctionUtility("1.5 if t<2.0 3.0", parameters)
 
+    def test_GenericFunctionUtilityError(self):
+        with self.assertRaisesRegex(Exception, 'Parsing error in function: \(0\)\*\(50\*\(expp\(t\)-1\)\)\nError occurred near here :             \^ \(char \[12\]\)\nCheck your locale \(e.g. if "." or "," is used as decimal point'):
```

Comment:
Note that with the wrong locale this test still passed in my computer, so this test might give wrong confidence about the issue. Tinyexpr only failed in the very specific circumstances I expose in my issue (the rom unit test).